README.md

```diff
@@ -1,9 +1,87 @@
 # Epub player library for Sunbird platform!
 Contains Epub player library components powered by angular. These components are designed to be used in sunbird consumption platforms *(mobile app, web portal, offline desktop app)* to drive reusability, maintainability hence reducing the redundant development effort significantly.
 
-# Getting Started
-For help getting started with a new Angular app, check out the [Angular CLI](https://angular.io/cli).
-If you have an Angular ≥ 9 CLI project, you could simply use our schematics to add sunbird-epub-player library to it.
+# Getting Started with different integrations steps
+ The Epub player can be integrated as web component in plain javascript projects and as web component in angular apps and also as angular library in angular and mobile applications.
+
+# Use as web components	
+
+Any web application can use this library as a web component. It accepts couple of inputs and triggers some events back to the application.
+
+Follow below-mentioned steps to use it in plain javascript project:
```

Comment:
This is not correct mention that use below steps to use as web component, they can use it anywhere

Reply:
Updated changes